code/modules/mob/living/simple_animal/abnormality/teth/crumbling_armor.dm

```diff
@@ -58,41 +53,123 @@
 		user.adjustBruteLoss(500)
 		return
 	if(work_type == ABNORMALITY_WORK_REPRESSION)
-		if (user in foolish_samurai_warrior)
-			switch (foolish_samurai_warrior[user])
-				if(1) // From Courage to Recklessness
-					playsound(get_turf(user), 'sound/machines/clockcult/stargazer_activate.ogg', 50, 0, 2)
-					user.cut_overlay(mutable_appearance(buff_icon, "courage", -ABOVE_MOB_LAYER))
-					user.adjust_attribute_buff(FORTITUDE_ATTRIBUTE, -5)
-					user.add_overlay(mutable_appearance(buff_icon, "recklessFirst", -ABOVE_MOB_LAYER))
-					to_chat(user, "<span class='userdanger'>Your muscles flex with strength!</span>")
-					foolish_samurai_warrior[user] = 2
-					return
-				if(2) // From Recklessness to Foolishness
-					playsound(get_turf(user), 'sound/machines/clockcult/stargazer_activate.ogg', 50, 0, 2)
-					user.cut_overlay(mutable_appearance(buff_icon, "recklessFirst", -ABOVE_MOB_LAYER))
-					user.adjust_attribute_buff(FORTITUDE_ATTRIBUTE, -5)
-					user.adjust_attribute_buff(JUSTICE_ATTRIBUTE, 5)
-					user.add_overlay(mutable_appearance(buff_icon, "recklessSecond", -ABOVE_MOB_LAYER))
-					to_chat(user, "<span class='userdanger'>You feel like you could take on the world!</span>")
-					foolish_samurai_warrior[user] = 3
-					return
-				if(3) // From Foolishness to Suicidal
-					playsound(get_turf(user), 'sound/machines/clockcult/stargazer_activate.ogg', 50, 0, 2)
-					user.cut_overlay(mutable_appearance(buff_icon, "recklessSecond", -ABOVE_MOB_LAYER))
-					user.adjust_attribute_buff(FORTITUDE_ATTRIBUTE, -10)
-					user.adjust_attribute_buff(JUSTICE_ATTRIBUTE, 5)
-					user.add_overlay(mutable_appearance(buff_icon, "foolish", -ABOVE_MOB_LAYER))
-					to_chat(user, "<span class='userdanger'>You are a God among men!</span>")
-					foolish_samurai_warrior[user] = 4
-					return
-				if(4) // You can progress no further down this fool-hardy path
-					return
-		else // Give Courage
+		if (src.icon_state == "megalovania")
```

Comment:
I don't know if I should hate you for this or not.

Reply:
You should.